ecmascript/parser/src/lexer/util.rs

```diff
@@ -268,7 +268,7 @@ pub trait CharExt: Copy {
             None => return false,
         };
         // TODO: Use Unicode ID instead of XID.
-        c == '$' || c == '_' || c == '\u{200c}' || c == '\u{200d}' || UnicodeXID::is_xid_continue(c)
```

Comment:
Why did you remove _?

Reply:
Beacuse `UnicodeXID::is_xid_continue('_')` will return `true`.